sql/merge.sql

```diff
@@ -10,4 +10,35 @@ select
      submitter_title as original_data_source
 from gencc;
 
+insert into gene_to_disease by name
+select
+    mim_gene_curie as original_gene_id,
+    gene_sssom.subject_id as hgnc_id,
+    disease_curie as original_disease_id,
+    mondo_sssom.subject_id as mondo_id,
+    -- derived from https://github.com/exomiser/Exomiser/blob/ffb2da103ca54a9de03ceecfc70c8ddfcebba3fd/exomiser-core/src/main/java/org/monarchinitiative/exomiser/core/prioritisers/model/InheritanceMode.java#L38
+    case
+        when moi_code = 'B' then 'exomiser:B' -- autosomal dominant/recessive ?
+        when moi_code = 'D' then 'HP:0000006' -- autosomal dominant
+        when moi_code = 'M' then 'HP:0001427' -- mitochondrial
+        when moi_code = 'P' then 'HP:0010982' -- polygenic
+        when moi_code = 'R' then 'HP:0000007' -- autosomal recessive
+        when moi_code = 'S' then 'exomiser:S' -- somatic ?
```

Comment:
Yes, B = Both AD and AR, S = Somatic

Reply:
Would it make sense to add HPO terms for those? 

I don't 100% remember what I was up to here, but I think I was shooting for making a converged table where everything can be aligned on HGNC + MONDO + HP MOI as the natural key. 

I might have that wrong though, maybe it makes more sense to use HGNC + MONDO as the natural key and we can just report the MOI provided from each source separately without them needing to be perfectly aligned.

Reply:
HPO terms for these map to `S=Somatic mosaicism=HP:0001442` and `B=Semidominant=HP:0032113`.

I think there might be some variability in the MOI terms, as with the strength assertion, so you best bet is probably the HGNC + MONDO combo.